documentation/spherex_data_products.md

```diff
@@ -78,6 +85,123 @@ HDU 6: WCS-WAVE
 
 - `level2_2025W18_2B_0001_1D1_spx_l2b-v12-2025-164.fits`
 
+## Cutouts of Spectral Image MEFs
```

Comment:
Do you mean to have all these code snippets in here? Or should we instead link to your new notebook?

Or I could also look into it if it's indeed easy to reference code sections from that in here.